scripts/warm-dotio-cache.ts

```diff
@@ -0,0 +1,92 @@
+import { URL, URLSearchParams } from 'url'
+import createFetch from '@vercel/fetch'
+import pMap from 'p-map'
+import proxyConfig from '../build-libs/proxy-config'
+
+interface StaticPathsResponse {
+  meta: {
+    status_code: number
+    status_text: string
+  }
+  result: {
+    paths: {
+      params: {
+        page: string[]
+      }
+    }[]
+  }
+}
+
+const fetch = createFetch()
+
+/**
+ * Returns a string representing a date that's `daysAgo` in the past.
+ * Format: YYYY-MM-DD HH:MM:SS
+ */
+function getDateRange(daysAgo: number): string {
+  const d = new Date()
+  d.setTime(d.getTime() - 24 * 60 * 60 * 1000 * daysAgo)
+  return d.toISOString().substring(0, 19).split('T').join(' ')
+}
```

Comment:
How stable is this? It would be nice if we could make the timestamp generic so we can leverage caching on the `static_paths` endpoint.

Reply:
We could omit the time portion, which would basically give us a stable value for 24 hours. So every request today would have resulted in `2022-04-06 00:00:00`

```js
`${d.toISOString().substring(0, 10)} 00:00:00`
```

Reply:
That seems like a good approach to me!